package.json

```diff
@@ -17,56 +17,52 @@
+    "postbuild:css:default": "postcss --verbose --config postcss.config.js -d dist/ dist/video-js.css",
     "build:lang": "npm-run-all build:lang:*",
     "build:lang:js": "vjslang --dir dist/lang",
-    "build:lang:copy": "shx cp -R lang/* dist/lang/",
-    "pretest:a11y": "shx cp sandbox/descriptions.html.example sandbox/descriptions.test-a11y.html",
-    "test:a11y": "node build/test-a11y.js",
-    "clean": "shx rm -rf ./dist ./test/dist build/temp/*",
-    "postclean": "shx mkdir -p ./dist/lang ./test/dist build/temp",
-    "changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
-    "build": "grunt dist",
-    "minify": "babel-node build/minify.js",
-    "cssmin": "npm-run-all cssmin:*",
-    "cssmin:cdn": "cleancss dist/alt/video-js-cdn.css -o dist/alt/video-js-cdn.min.css",
-    "cssmin:default": "cleancss dist/video-js.css -o dist/video-js.min.css",
-    "sass": "npm-run-all sass:* autoprefixer",
-    "sass:cdn": "node-sass src/css/vjs-cdn.scss dist/alt/video-js-cdn.css",
-    "sass:default": "node-sass src/css/vjs.scss dist/video-js.css",
-    "autoprefixer": "npm-run-all -p autoprefixer:*",
-    "autoprefixer:main": "postcss --verbose --config postcss.config.js -d dist/ dist/video-js.css",
-    "autoprefixer:cdn": "postcss --verbose --config postcss.config.js -d dist/alt dist/alt/video-js-cdn.css",
-    "rollup": "rollup -c",
-    "rollup-dev": "rollup -c -w",
+    "build:lang:copy": "shx --verbose cp -R lang/* dist/lang/",
+    "minify": "npm-run-all minify:*",
+    "minify:js": "babel-node build/minify.js",
+    "minify:css": "npm-run-all minify:css:*",
+    "minify:css:cdn": "cleancss dist/alt/video-js-cdn.css -o dist/alt/video-js-cdn.min.css",
+    "minify:css:default": "cleancss dist/video-js.css -o dist/video-js.min.css",
     "watch": "npm-run-all -p watch:*",
     "watch:lang": "chokidar --initial 'lang/**/*.json' -c 'npm run build:lang'",
-    "watch:rollup": "npm run rollup-dev",
-    "watch:sass-default": "npm run sass:default -- --watch 'src/**/**/*.scss'",
-    "watch:sass-cdn": "npm run sass:cdn -- --watch 'src/**/**/*.scss'",
+    "watch:rollup": "rollup -c -w ",
+    "watch:css-default": "npm run build:css:default -- --watch 'src/**/**/*.scss'",
+    "watch:css-cdn": "npm run build:css:cdn -- --watch 'src/**/**/*.scss'",
     "assets": "node build/assets.js",
-    "grunt": "grunt",
     "lint": "vjsstandard",
-    "start": "npm-run-all -p start:grunt watch",
-    "start:grunt": "grunt dev",
+    "lint-errors": "vjsstandard --errors",
+    "start": "npm-run-all -p watch karma-server",
     "karma-server": "karma start test/karma.conf.js --singleRun=false --auto-watch",
-    "karma-run": "karma start test/karma.conf.js",
-    "test": "grunt test",
-    "docs": "npm run docs:lint && npm run docs:api",
-    "jsdoc": "jsdoc",
-    "predocs:api": "node -e \"var s=require('shelljs'),d=['docs/api'];s.rm('-rf',d);\"",
-    "checktranslations": "grunt check-translations",
+    "test": "npm-run-all build-dev test:*",
```

Comment:
what about moving `build-dev` to `pretest`, then running `test:*` in parallel?

Reply:
makes sense